src/js_sync.hpp

```diff
@@ -321,6 +330,8 @@ void SessionClass<T>::get_config(ContextType ctx, ObjectType object, ReturnValue
         ObjectType config = Object::create_empty(ctx);
         Object::set_property(ctx, config, "user", create_object<T, UserClass<T>>(ctx, new User<T>(session->config().user, nullptr))); // FIXME: nullptr is not an app object
         // TODO: add app id
+        bson::Bson partition_value_bson = bson::parse(session->config().partition_value);
```

Comment:
why do we do a roundtrip to/from Bson. Can't we just use the `partition_value` and set it to `Object::set_property(ctx, config, "partitionValue"`

Reply:
`session->config().partition_value` is a serialized bson/ejson string (I believe this is a common interface for all SDKs to OS/Sync (?)), we could store the original value somewhere - but after going through the different approaches with @kneth, I went with parsing the value used by Sync.

Reply:
👍